dev-python/dkimpy/dkimpy-1.0.4.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="DKIM and ARC email signing and verification library"
+HOMEPAGE="https://launchpad.net/dkimpy"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE=""
+
+DEPEND="
+	dev-python/dnspython[${PYTHON_USEDEP}]
+"
+RDEPEND="${DEPEND}"
```

Comment:
Do you really need it outside `test?`, and then `disutils_enable_tests` takes care of adding it.

Reply:
It is listed in install_requires so I figured it must be there ?

Reply:
It is actually needed in RDEPEND